fastlane/lib/fastlane/swift_lane_manager.rb

```diff
@@ -33,7 +33,7 @@ def self.cruise_lane(lane, parameters = nil, env = nil, disable_runner_upgrades:
         # wait on socket_thread to be in ready state, then start the runner thread
         self.cruise_swift_lane_in_thread(lane, parameters, swift_server_port)
 
-        socket_thread.join
+        socket_thread.value
```

Comment:
Do you know why this change was needed? Just curious 😊

Reply:
I added it to not just wait, but also get the output from the thread.
On verbose mode, it allows output from the runner thread, so you can see traces and ease feedback from fastlane swift thread.
You could see the change isolated [here](https://github.com/fastlane/fastlane/pull/19555/commits/1e15cdcffecea2e7de09689d5d91d0b30a2db647) with the  other verbose control flag changes.